bindings/python/src/mesh/builder/regular_grid_builder.h

```diff
@@ -38,6 +38,11 @@
                 const Point##dimension##D&, std::array< index_t, dimension >,  \
                 std::array< double, dimension > ) )                            \
                 & RegularGridBuilder##dimension##D::initialize_grid )          \
+        .def( "initialize_grid",                                               \
```

Comment:
You have to give a different name than the above initialization, as python functions cannot be overloaded